frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,747 @@
+            <svg
+            key={star}
+            onClick={() => setRating(star)}
+            onMouseEnter={() => setHoverRating(star)}
+            onMouseLeave={() => setHoverRating(0)}
+            xmlns="http://www.w3.org/2000/svg"
+            className={`w-6 h-6 cursor-pointer ${
+                (hoverRating || rating) >= star ? "text-yellow-400" : "text-gray-300"
+            }`}
+            fill="currentColor"
+            viewBox="0 0 20 20"
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)